schema_based_on_diagram.sql

```diff
@@ -0,0 +1,79 @@
+/* Create Tables */
+
+CREATE TABLE invoice_items (
+  id SERIAL PRIMARY KEY,
+  unit_price DECIMAL,
+  quantity INTEGER,
+  total_price DECIMAL,
+  invoice_id INTEGER,
+  treatment_id INTEGER
+)
+
+
+CREATE TABLE treatments (
+  id SERIAL PRIMARY KEY,
+  type VARCHAR(50),
+  name VARCHAR(100)
+)
+
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP,
+  patient_id INTEGER,
+  status VARCHAR(50)
+)
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(100),
+  date_of_birth DATE
+)
+
+CREATE TABLE invoices (
+  id SERIAL PRIMARY KEY,
+  total_amount DECIMAL,
+  generated_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_history_id INTEGER
+)
+
+/* Create the junction table for medical_histories and treatments */
+
+ALTER TABLE invoice_items
+ADD FOREIGN KEY (treatment_id) REFERENCES medical_histories(id);
+
+-- Indexes
+CREATE INDEX medical_histories_id_index ON medical_histories_and_treatments_junction_table(medical_histories_id DESC);
+CREATE INDEX treatments_id_index ON medical_histories_and_treatments_junction_table(treatments_id DESC);
+
```

Comment:
- [ ] [REQUIRED] Please implement [previous code reviewer](https://github.com/mkaya13/patients_db/pull/1#discussion_r1085417904)'s suggestion because you forgot to add `CREATE INDEX` for other `3` foreign keys. All `CREATE INDEX` should be `5`, please implement it 🙏🏼